crates/store/re_redap_client/src/grpc.rs

```diff
@@ -517,39 +496,55 @@ async fn stream_partition_from_server(
     let store_id = store_info.store_id.clone();
 
     if tx
-        .send(LogMsg::SetStoreInfo(SetStoreInfo {
-            row_id: *re_chunk::RowId::new(),
-            info: store_info,
-        }))
+        .send(
+            LogMsg::SetStoreInfo(SetStoreInfo {
+                row_id: *re_chunk::RowId::new(),
+                info: store_info,
+            })
+            .into(),
```

Comment:
I believe this should work too

```suggestion
            SetStoreInfo {
                row_id: *re_chunk::RowId::new(),
                info: store_info,
            }
            .into(),
```